security/pkg/k8s/chiron/utils.go

```diff
@@ -85,19 +88,42 @@ func GenKeyCertK8sCA(certClient certclient.CertificateSigningRequestInterface, d
 		log.Errorf("CSR generation error (%v)", err)
 		return nil, nil, nil, err
 	}
-
-	// 2. Submit the CSR
 	csrName := getRandomCsrName(secretName, secretNamespace)
+	csrSpec := &cert.CertificateSigningRequestSpec{
+		Request: csrPEM,
+		Groups:  []string{"system:authenticated"},
```

Comment:
Interesting - what are the implications here ? That the cert will also be usable for k8s access ? That's new - should be explicitly stated.